content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -1,9 +1,79 @@
 # Setup a Google Drive DVC Remote
 
-Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
-[remote storage](/doc/command-reference/remote).
+In this guide we explain the existing ways to setup Google Drive
+[remote storage](/doc/command-reference/remote) for your <abbr>DVC
+projects</abbr>, along with the different benefits each one brings.
 
-## Configure a Google Cloud project
+Note that to start using a GDrive remote, you only need to add it with a valid
+URL (see next section), and use any DVC command that requires it (see
```

Comment:
see next section -> see below how to specify it for shared drive, shared folders, etc ? be a bit more explicit why would I need to take a look there

Reply:
I agree with giving more motivation on URL formation, but I added it in the URL format section and just linked from here instead. See 9d9ebec4. Good?

Reply:
kk, it's fine, but see other comment about the basic case ... it feels that it's wrong that we use `root` and if we switch to something like `gdrive://FOLDER_ID/path` section that explains URLs becomes even more important. And explaining limitations for different URLs becomes very important (e.g. for `grdive://root` it won't be easy to share, and can be even dangerous since `gdrive://root` will be resolved fine and multiple people will push data to two different GD accounts)

Reply:
Yes, good points. Will address along with what I summarized in https://github.com/iterative/dvc.org/pull/1096#pullrequestreview-386926510.